arches/app/models/migrations/11799_require_graph_slug.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 5.2b1 on 2025-03-19 11:14
+
+import django.core.validators
+import re
+from django.db import migrations, models
+from slugify import slugify
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("models", "11882_principal_user_read_only"),
+    ]
+
+    def add_missing_graph_slug(apps, schema_editor):
+        Graph = apps.get_model("models", "GraphModel")
+        graphs_missing_slug = Graph.objects.filter(slug__isnull=True)
+        for graph in graphs_missing_slug:
+            graph.slug = slugify(str(graph.name), separator="_")
```

Comment:
Names aren't necessarily unique, so I can envision this failing. Maybe we want to use `add_slug()` here? But then we would need to set `use_in_migrations = True` on the GraphManager. (And that means not using the shortcut `objects = GraphQuerySet.as_manager()` anymore, following the second example [here](https://docs.djangoproject.com/en/5.1/topics/migrations/#model-managers).) Maybe you'd prefer a simpler solution just inline here.

Reply:
Just tried to sketch this out and didn't get it working, so this is probably way overkill. Let's do a simpler solution of just avoiding using any existing slugs.